python/rikai/spark/sql/codegen/fs.py

```diff
@@ -13,18 +13,18 @@
 #  limitations under the License.
 
 import secrets
-from typing import Callable, IO, Any, Dict, Mapping, Optional, Union
+from typing import IO, Any, Callable, Dict, Mapping, Optional, Union
```

Comment:
it is weird that IO is before Any

Reply:
i'm turning off `order_by_type` (isort thinks IO is a constant)